controlpanel/frontend/static/javascripts/modules/tool-status.js

```diff
@@ -21,9 +24,32 @@ moj.Modules.toolStatus = {
   buildEventHandler(listener) {
     return event => {
       const data = JSON.parse(event.data);
-      if (data.toolName == listener.dataset.toolName) {
-        listener.innerText = data.status;
+      if (data.toolName != listener.dataset.toolName) {
+        return;
+      }
+      listener.querySelector(this.statusLabelClass).innerText = data.status;
+      switch (data.status.toUpperCase()) {
```

Comment:
AFAICT `data.status` is already sent in uppercase so probably `toUpperCase()` could be removed.

Reply:
It definitely isn't sent in uppercase.